vitrina/requests/migrations/0002_auto_20221010_1418.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 3.2.16 on 2022-10-10 11:18
+
+import collections
+
+from django.db import migrations
+
+
+def create_history_objects(apps, schema_editor):
+    RequestEvent = apps.get_model('vitrina_requests', 'RequestEvent')
+    Request = apps.get_model('vitrina_requests', 'Request')
+    Revision = apps.get_model('reversion', 'Revision')
+    Version = apps.get_model('reversion', 'Version')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    User = apps.get_model('vitrina_users', 'User')
+
+    users = collections.defaultdict(dict)
+    for user in User.objects.filter(organization_id__isnull=False):
+        org = user.organization_id
+        user_name = f'{user.first_name} {user.last_name}'
+        users[org][user_name] = user
+
+    for event in RequestEvent.objects.all():
+        if event.request_id:
+            content_type = ContentType.objects.get_for_model(Request)
+            request = Request.objects.get(pk=event.request_id)
+            revision = Revision.objects.create(
+                date_created=event.created,
+                comment=event.type or "",
+                user=users.get(request.organization_id, {}).get(event.meta)
```

Comment:
Added one case, to take user from `request.meta`, but additionally on `create` action, user should be taken from `request.user`. We don't want to lose this information.